packages/components/grid/src/Col.vue

```diff
@@ -0,0 +1,137 @@
+  if (isNumber(flex)) {
+    return `${flex} ${flex} auto`
+  }
+
+  if (/^\d+(\.\d+)?(px|em|rem|%)$/.test(flex)) {
+    return `0 0 ${flex}`
+  }
+
+  return flex
+}
+
+const numberGt0 = (n: unknown) => isNumber(n) && n > 0
+
+const normalizeSizeProps = (rawProps: number | ColBreakpointConfig) => {
+  let result: ColBreakpointConfig = {}
+  if (isNumber(rawProps)) {
+    result.span = rawProps
+  } else if (isObject(rawProps)) {
+    result = rawProps || {}
+  }
+  return result
+}
+
+const getAllBreakpointCls = (props: ColProps, colCls: string) => {
+  let allBreakpointCls = {}
+
+  AbbrBreakpoints.forEach(size => {
+    const sizeProps: ColBreakpointConfig = normalizeSizeProps(props[size] || {})
+
+    allBreakpointCls = {
+      ...allBreakpointCls,
+      [`${colCls}-${size}-${sizeProps.span}`]: !isUndefined(sizeProps.span),
+      [`${colCls}-${size}-order-${sizeProps.order}`]: numberGt0(sizeProps.order),
+      [`${colCls}-${size}-offset-${sizeProps.offset}`]: numberGt0(sizeProps.offset),
+      [`${colCls}-${size}-push-${sizeProps.push}`]: numberGt0(sizeProps.push),
+      [`${colCls}-${size}-pull-${sizeProps.pull}`]: numberGt0(sizeProps.pull),
+    }
+  })
+  return allBreakpointCls
+}
+
+export default defineComponent({
+  name: 'IxCol',
+  props: {
+    span: PropTypes.number,
+    order: PropTypes.number,
+    offset: PropTypes.number,
+    push: PropTypes.number,
+    pull: PropTypes.number,
+    xs: breakpointConfig,
+    sm: breakpointConfig,
+    md: breakpointConfig,
+    lg: breakpointConfig,
+    xl: breakpointConfig,
+    flex: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  },
+  setup(props: ColProps) {
+    const classNamePrefix = 'ix'
+    const gutter = inject<Ref<[number, number]>>('rowGutter')
+
+    const classes = useClasses(props, classNamePrefix)
+    const style = useColStyle(props, gutter)
+
+    return {
+      classes,
+      style,
+    }
+  },
+})
+
+function useClasses(props: ColProps, prefix: string) {
+  return computed(() => {
+    const colCls = `${prefix}-col`
+    const allBreakpointCls = getAllBreakpointCls(props, colCls)
+
+    return {
+      [colCls]: true,
+      [`${colCls}-${props.span}`]: !isUndefined(props.span),
+      [`${colCls}-order-${props.order}`]: numberGt0(props.order),
+      [`${colCls}-offset-${props.offset}`]: numberGt0(props.offset),
+      [`${colCls}-push-${props.push}`]: numberGt0(props.push),
+      [`${colCls}-pull-${props.pull}`]: numberGt0(props.pull),
+      ...allBreakpointCls,
+    }
+  })
+}
+
+function useColStyle(props: ColProps, gutter: Ref<[number, number]> | undefined) {
+  const getPaddingNum = (n: number) => n / 2
+
+  return computed(() => {
+    const style: CSSProperties = {}
+
+    if (gutter?.value) {
+      const horizontalGutter = gutter.value[0]
+      const verticalGutter = gutter.value[1]
+
+      if (horizontalGutter > 0) {
+        style.paddingLeft = `${getPaddingNum(horizontalGutter)}px`
```

Comment:
```suggestion
        style.paddingLeft = `${horizontalGutter / 2 }px`
```
更直观，我刚刚找 `getPaddingNum` 找了半天。。。